### ISSUE TITLE
Wrong intendation in vbs

### ISSUE THREAD

**Issue by a0bad** [akalongman/sublimetext-codeformatter#389]
Hello,
there is a problem using codeformatter with vbscript. The intendation of "opening tags" is not as expected. it should be aligned with the "closing tag".

I am using the default settings for vbscript.

thanks for the awesome plugin!

![image](https://user-images.githubusercontent.com/1195894/54761106-c1139b80-4bf1-11e9-9e77-89223e499bf3.png)

**Comment by viewsafe** [comment]
yes, as above pictures, the wrong intendation in VBS, need solve this issue, is there any update?
